tracer/test/test-applications/integrations/Samples.GrpcDotNet/Samples.GrpcDotNet.csproj

```diff
@@ -17,6 +17,11 @@
     <GeneratedFolder>Generated_Code\$(TargetFramework)</GeneratedFolder>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(IsOSX)' == 'true'">
+    <!-- If OSX we force the x64 version (protoc is only available in x64) -->
+    <Protobuf_ToolsCpu>x64</Protobuf_ToolsCpu>
+  </PropertyGroup>
```

Comment:
Urgh, yuk, this is the reason we have all that garbage in the Directory.Build.props?! 🤮 😅

If so, I would probably just move the condition here
```suggestion
  <PropertyGroup Condition="'$([System.Runtime.InteropServices.RuntimeInformation]::IsOSPlatform($([System.Runtime.InteropServices.OSPlatform]::OSX)))' == 'true'">
    <!-- If OSX we force the x64 version (protoc is only available in x64) -->
    <Protobuf_ToolsCpu>x64</Protobuf_ToolsCpu>
  </PropertyGroup>
```

I understand wanting to "generalize" it, but we have like 25 Directory.Build.props files... only having that variable available in some of them seems like it might be more confusing!